Assets/Scripts/Background/BeyondBorderResize.cs

```diff
@@ -0,0 +1,118 @@
+// Thanks to Loek van den Ouweland for scaling code explanation;
+// their article is where I came across a script with similar code for obtaining camera values and scaling a background.
+// van den Ouweland, L. (2019, October 9). STRETCH A UNITY SPRITE TO FILL THE SCREEN IN A 2D GAME (GAMEOBJECT, NOT UI-CANVAS). 
+// Loek van den Ouweland Software Engineering. Retrieved March 20, 2023, from 
+// https://www.loekvandenouweland.com/content/stretch-unity-sprite-to-fill-the-screen.html.
+// This code increases/decreases the size of the GameObject sprite so that it can cover the empty left/right background space that exists 
+// when the player is using a large/horizontal screen to play the game. The sizing is dependent on the distance between the assigned border's 
+// corresponding side (boolean right) and the same camera border side. NOTE: This script must be applied to 2 filler sprites (right and left) to 
+// fill all empty space to the left and right of the game screen. This script was created for tiled continuous sprites in mind, which is why 
+// the size of the sprite is altered, not the scale.
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BeyondBorderResize : MonoBehaviour
+{
+    private float cameraHeight;
+    private float cameraWidth;

```

Comment:
I think that `currentSize.x` and `currentSize.y` should be used instead of `cameraWidth` and `cameraHeight` to reduce the number of instance variables here. You can also use `fillerSprite.size.x` and `fillerSprite.size.y` instead of having `spriteWidth` and `spriteHeight`.

Reply:
Got it, "cameraHeight" is now a local variable in the "Awake" and "Update" functions, just so that the assignment of "currentCameraSize" and "newCameraSize" is still concise. "cameraWidth", "spriteWidth", and "spriteHeight" have been removed, as well.

Reply:
Yes, having a local `cameraHeight` variable is a good way to avoid duplicating multiplication when getting the camera width.